serve/plugin.go

```diff
@@ -306,12 +268,7 @@ func (s *PluginServe) newCmdPluginServe() *cobra.Command {
 	cmd.Flags().StringVar(&otelEndpointURLPath, "otel-endpoint-urlpath", "", "Open Telemetry HTTP collector endpoint URL path")
 	cmd.Flags().StringArrayVar(&otelEndpointHeaders, "otel-endpoint-headers", []string{}, "Open Telemetry HTTP collector endpoint headers")
 	cmd.Flags().BoolVar(&otelEndpointInsecure, "otel-endpoint-insecure", false, "use Open Telemetry HTTP endpoint (for development only)")
-	cmd.Flags().BoolVar(&noSentry, "no-sentry", false, "disable sentry")
```

Comment:
I'm not sure here tbh what would be the best approach, whether to remove this or have it do nothing and mark as deprecated. Wdyt?

Reply:
I would keep the option but make it deprecated and no-op.

Reply:
That makes sense, I wasn't sure if deleting it was the best decision as if it was like that, I'd have had to delete it from the other plugin sdks as well.

Reply:
Yeah I think if we remove the flag new plugins might not work with older CLI versions (don't remember if we're doing strict arg parsing)